provisioner/roles/manage_ec2_instances/tasks/addhost_security_automation.yml

```diff
@@ -0,0 +1,93 @@
+- name: grab facts for splunknode (SECURITY AUTOMATION MODE)
+  ec2_instance_facts:
+    region: "{{ ec2_region }}"
+    filters:
+      instance-state-name: running
+      "tag:Workshop_splunk": "{{ec2_name_prefix}}-splunk"
+  register: splunk_node_facts
+
+- name: grab facts for snort node (SECURITY AUTOMATION MODE)
+  ec2_instance_facts:
+    region: "{{ ec2_region }}"
+    filters:
+      instance-state-name: running
+      "tag:Workshop_snort": "{{ec2_name_prefix}}-snort"
+  register: snort_node_facts
+
+- name: grab facts for checkpoint node (SECURITY AUTOMATION MODE)
+  ec2_instance_facts:
+    region: "{{ ec2_region }}"
+    filters:
+      instance-state-name: running
+      "tag:Workshop_checkpoint": "{{ec2_name_prefix}}-checkpoint"
+  register: checkpoint_node_facts
+
+- name: grab facts for rtr4 node (SECURITY AUTOMATION MODE)
```

Comment:
What is this for?

Reply:
that is a left over from something that didn't pan out ... will clean up